build/integration/windows/Jenkinsfile

```diff
@@ -0,0 +1,53 @@
+pipeline {
+  agent {
+    label { label 'windows-build01 || windows-build02 || windows-build03' }
```

Comment:
not working with 04 - black screen, no idea why

Reply:
maybe other setup or someone changed something. or logged in per rcp?

Reply:
I will restart 04 and double check it

Reply:
restart did not solve it, I leave that as it is for now ;)